### DISCUSSION TITLE
WT, COMMON_LVB_UNDERSCORE, WriteConsoleOutput?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#14053] [General]
The short question is "Should it work?".  It doesn't seem to.  It works in a console.

**Comment by DHowett** [comment]
They "should," but they do not yet. We're tracking that over in https://github.com/microsoft/terminal/issues/8037

**Comment by vefatica** [comment]
Thanks.  That's old.  I won't be holding my breath.
